### PR TITLE
Erfan cubemap render

### PR DESCRIPTION
**Pull request by Erfan-Ahmadi**

### CODE REVIEW

File: examples_tests/22.RaytracedAO/main.cpp [hunk 1]
```diff
@@ -584,6 +584,12 @@ int main(int argc, char** argv)
 				nbl::core::vectorSIMDf(0, 0, +1, 0), // +Z
 				nbl::core::vectorSIMDf(0, 0, -1, 0), // -Z
 			};
+
+			if(!mainSensorData.rightHandedCamera)
+			{
+				camViews[0] *= -1;
+				camViews[1] *= -1;
+			}
```
Comment (devshgraphicsprogramming) [inline examples_tests/22.RaytracedAO/main.cpp:592]: @Erfan-Ahmadi you tested that it works?

File: examples_tests/22.RaytracedAO/pannelum_sample_config.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    "type": "cubemap",
+    "cubeMap": [
+        "Render_scene_z-_denoised.png",
+        "Render_scene_x+_denoised.png",
+        "Render_scene_z+_denoised.png",
+        "Render_scene_x-_denoised.png",
+        "Render_scene_y+_denoised.png",
+        "Render_scene_y-_denoised.png"
+    ]
+}
```
Comment (devshgraphicsprogramming) [inline examples_tests/22.RaytracedAO/pannelum_sample_config.json:11]: dont commit this file

File: examples_tests/22.RaytracedAO/main.cpp (hunk 1 above)
Comment (Erfan-Ahmadi) [reply examples_tests/22.RaytracedAO/main.cpp:592]: Yea.

File: examples_tests/22.RaytracedAO/pannelum_sample_config.json (hunk 2 above)
Comment (Erfan-Ahmadi) [reply examples_tests/22.RaytracedAO/pannelum_sample_config.json:11]: I'll delete.

**Pull request merged**